tests/Client/GusApiClientTest.php

```diff
@@ -21,76 +25,98 @@
 use GusApi\Type\Response\SearchDataResponse;
 use GusApi\Type\Response\SearchResponseCompanyData;
 use GusApi\Type\Response\SearchResponseRaw;
-use GusApi\Type\SearchParameters;
 use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
+use SoapClient;
 
-class GusApiClientTest extends TestCase
+final class GusApiClientTest extends TestCase
 {
-    /**
-     * @var GusApiClient
-     */
-    protected $gusApiClient;
+    use GetContentTrait;
 
-    /**
-     * @var MockObject
-     */
-    protected $soap;
+    private GusApiClient $gusApiClient;
+    /** @var SoapClient|MockObject */
+    private SoapClient|MockObject $soap;
 
     public function setUp(): void
     {
-        $this->soap = $this->getMockFromWsdl(__DIR__.'/../UslugaBIRzewnPubl.xsd');
-
+        $this->soap = $this->getMockFromWsdl(__DIR__ . '/../UslugaBIRzewnPubl.xsd', SoapClient::class);
```

Comment:
This fixes the failing tests:
```suggestion
        $this->soap = $this->getMockFromWsdl(__DIR__ . '/../UslugaBIRzewnPubl.xsd');
```

Reply:
Probably disable call `$callOriginalConstructor = false` should fix this problem